src/molecules/tabAction/TabAction.js

```diff
@@ -27,4 +28,4 @@ const TabAction = (props) => {
   );
 };
 
-export default TabAction;
+export default React.memo(TabAction);
```

Comment:
Why do you think that React.memo will improve this functionality?

Reply:
in this case, 
the component tree will be like this
PricingPage
  -----Pricing Plans
     --------Tab Action

in pricing plans component we have state for selected pricing card, so whenever we click a price plan we will re-evaluated the pricing plans component and its child ( TabAction ). To get better performance, the React.Memo will prevent the child being re evaluated if there is no changes for the props that is being passed to the component